database/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -15,6 +15,7 @@ import slick.driver.JdbcProfile
 import scala.concurrent.duration._
 import scala.concurrent.{Await, ExecutionContext, Future}
 
+
```

Comment:
This is how I roll, putting two spaces between the import preamble and the meat of the code that follows.  There are probably dozens of files like this.